### DISCUSSION TITLE
Change Growl.Ask buttons

### DISCUSSION THREAD

**Discussion by compil3** [HandyOrg/HandyControl#1547] [Q&A]
Currently they display in Chinese.
Is there a way to change them?

**Comment by ghost1372** [comment]
> Currently they display in Chinese.
> Is there a way to change them?

Install language packages or use custom version which default language is english

**Comment by Angeloasl20** [comment]
Hi, I am Brazilian and I'm having trouble with the same thing. The buttons display in chinese, I downloaded the "HandyControl.Lang.pt-BR by HandyOrg" nugget package and I added the following code:

        protected override void OnStartup(StartupEventArgs e)
        {
            base.OnStartup(e);

            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
            CultureInfo.DefaultThreadCurrentCulture = CultureInfo.InvariantCulture;
            ConfigHelper.Instance.SetLang("pt-br");
            Lang.Culture = new CultureInfo("pt-br");
        }
        
 On my App.xaml.cs
 
 I have both "HandyControl by HandyOrg" and "HandyControl.Lang.pt-BR by HandyOrg" on version 3.5.1
 What should I do?
